src/alt.js

```diff
@@ -34,6 +34,13 @@ function uid(container, name) {
   return key
 }
 
+/* for now just detect whether it's instance of Immutable.js */
+function isImmutable(state) {
+  // let's check with that for now
+  // https://github.com/facebook/immutable-js/issues/421
+  return typeof state.toJS === 'function'
```

Comment:
In theory I like this, but I am wondering if we want to bake this into alt, as it relies on the api of a specific library, one that is not even an alt dependency.

It also seems weird to me to support just one library when there are so many immutable libraries popping up these days.